test/rules/CloudFront.test.ts

```diff
@@ -284,10 +290,28 @@ describe('Amazon CloudFront', () => {
     });
   });
 
-  describe('CloudFrontDistributionS3OriginAccessIdentity: CloudFront distributions use an origin access identity for S3 origins', () => {
+  describe('CloudFrontDistributionS3OriginAccessIdentity: CloudFront Streaming distributions use an origin access identity for S3 origins', () => {
     const ruleId = 'CloudFrontDistributionS3OriginAccessIdentity';
-    test('Noncompliance 1', () => {
-      new CfnDistribution(stack, 'rDistribution', {
+    test('Noncompliance', () => {
+      new CfnStreamingDistribution(stack, 'rStreamingDistribution', {
```

Comment:
Please the remove the `r` prefix from the ids, it's a naming convention that doesn't work well with CDK and that I'd like to remove from the project

Reply:
fixed!